docs/docs/styled-components.md

```diff
@@ -20,7 +20,7 @@ First, open a new terminal window and run the following to create a new site:
 gatsby new styled-components-tutorial https://github.com/gatsbyjs/gatsby-starter-hello-world
 ```
 
-Next, we'll install the necessary dependencies for `styled-components`, including the Gatsby plugin.
+Second, we'll install the necessary dependencies for `styled-components`, including the Gatsby plugin.
```

Comment:
Since this line is being updated, can you remove `we'll` here as well? Even though it was in there already :)

Reply:
Removed `we will` from all the styling docs. :)